src/sass/_shame-wizard.scss

```diff
@@ -0,0 +1,225 @@
+// This file is a place where you put all the code you're not proud of,
```

Comment:
I like this thought, I will just need to remember to clean up the unused styles in _shame.scss if this change gets approved.